src/index.html

```diff
@@ -9,9 +9,191 @@
+          >
+        </label>
+
+        <label class="label">
+          How old are You?
+
+          <input
+            type="number"
+            name="age"
+            placeholder="Your age"
+            value="12"
+            min="1"
+            max="100"
+            required
+          >
+        </label>
+
+        <label class="label">
+          Full date of birth:
+
+          <input
+            type="date"
+            name="date of birth"
+            placeholder="Date of birth"
+            required
+          >
+        </label>
+
+        <label>
+          I accept the term of the agreement
+
+          <input
+            type="checkbox"
+            name="terms"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label class="label">
+          E-meil:
+
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label>
+          Password:
+
+          <input
+            type="password"
+            name="password"
+            autocomplete="off"
+            required
+            minlength="8"
+            maxlength="30"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="label">
+          Do you love cats?
+
+          <label>
+            <input
+              type="radio"
+              name="cat"
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="cat"
+              value="no"
+              checked
+            >
+            No
+          </label>
+        </div>
+
+        <label class="label">
+          What is your favorite color?
+
+          <input
+            type="color"
+            name="favorite_color"
+          >
+        </label>
+
+        <label class="label">
+          What time do you go to bed?
+
+          <input
+            type="time"
+            name="your time go to bed"
+          >
+        </label>
+
+        <label class="label">
+          What are your favorite brand of cars?
+
+          <select name="brand_of_cars" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label>
+          How do you rate our work?
+
+          <input
+            type="range"
+            name="rate"
+            value="0"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <label class="label">
+          Comments:
+
+          <textarea
```

Comment:
checklist p.8:  Check the textarea behaviour - when you click on the textarea, the coursor should appear on the beginning of the input.